packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.css

```
@@ -2,6 +2,7 @@
   display: flex;
   flex-direction: row;
   min-width: 0;
+  color: var(--brand-text-link, #0176d3);
```

Apparently this one is actually camelcase
```suggestion
  color: var(--lwc-brandTextLink, #0176d3);
```
I have no idea why some are snake and some are camel. It's a zoo ¯\_(ツ)_/¯

---

`--lwc-brandTextLink` keebab-case plus camelCase 😱